specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2017-03-01/apimbackends.json

```diff
@@ -307,6 +307,57 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/backends/{backendid}/reconnect": {
+      "post": {
+        "tags": [
+          "Backends"
+        ],
+        "operationId": "Backend_Reconnect",
+        "description": "Notifies the APIM proxy to create a new connection to the backend after the specified timeout. If no timeout was specified, timeout of 2 minutes is used.",
+        "x-ms-examples": {
+          "ApiManagementBackendReconnect": {
+            "$ref": "./examples/ApiManagementBackendReconnect.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/BackendIdParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "$ref": "#/definitions/BackendReconnectContract"
+            },
+            "description": "Reconnect request parameters."
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
```

Comment:
Please add the final response of this API under "responses". It will allow customers know the final response.

Reply:
@ravbhatnagar this action isn't supposed to return final response. We don't return the status of the operation for this action.

Reply:
@promoisha - what does it mean to not return the final response of an operation. What you are saying means, the client will keep polling on this operation? How do you know the operation has completed? 200 OK or something else?

Reply:
@ravbhatnagar because there's no such thing as 'operation completed' in this case. This is not a long-running operation and since customer specifies a time period after which he wants new connections to be created, there's no need in polling.

Reply:
@promoisha - How would the customer know whether the reconnect succeeded or failed? What if after 2 minutes the service tried to reconnect but there was a failure? New connection created is an operation which must either succeed or fail. How can it not have a thing as "operation completed"?

Reply:
@ravbhatnagar    202 is a successful response.
> 6.3.  Successful 2xx
>
>   The 2xx (Successful) class of status code indicates that the client's
>   request was successfully received, understood, and accepted.
>   https://tools.ietf.org/html/rfc7231#section-6.3

HTTP makes no requirements on a server to provide a representation that describes the outcome of the accepted request.

Reply:
@darrelmiller - well,  202 is a successful response but it also indicates the processing has not completed. From the RFC link you shared this is what it says - 

"The 202 (Accepted) status code indicates that the request has been accepted for processing, 
but the processing has not been completed. ............ The entity returned with this response 
 SHOULD include an indication of the request's current status and either a pointer to a status 
 monitor or some estimate of when the user can expect the request to be fulfilled."

As for ARM, if it sees a 202 response without a location header, it will log a provider error against the RP and return the 202 to the client.  Don't know what the client will do with this 202. 
Anyway, small issue if you had like to fix.

Reply:
@ravbhatnagar  Returning a 202 to the client would be the desired behavior in this case.  We should review why ARM would consider a 202 without a Location header to be an error. In the text from the spec you quoted it says that returning a pointer to status monitor is only one option.
   
Hey, @fearthecowboy how would the client SDK react to getting a 202 response back, assuming the 202 has been documented as the only successful response in the OpenAPI description?

Reply:
If you're not using `long-running-operation`, the client shouldn't interpret the 202 as a hint to do polling, merely a 'Yeah, ok' response.

Reply:
We're not using it, so we should be fine here.